index.html

```diff
@@ -4,484 +4,528 @@
-                  />
-                  <input
-                    class="location-input"
-                    type="text"
-                    placeholder="Location"
-                    name="text"
-                    value=""
-                  />
-                </form>
+        <!-- Navigation menu -->
+        <nav class="nav-menu">
+          <ul class="nav-menu__list">
+            <li class="nav-menu__list-item"><a href="#home">Home</a></li>
+            <li class="nav-menu__list-item">
+              <a href="#categories">Categories</a>
+            </li>
+            <li class="nav-menu__list-item"><a href="#service">Service</a></li>
+            <li class="nav-menu__list-item"><a href="#jobs">Jobs</a></li>
+            <button class="button nav__button" href="#">Post a vacancy</button>
+          </ul>
+        </nav>
+        <nav class="menu nav-menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#home">Home</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#categories">Categories</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#service">Service</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#jobs">Jobs</a>
+            </li>
+          </ul>
+        </nav>
 
-                <!-- Search button -->
-                <div class="hero-section__button">
-                  <a class="buttons buttons__search-button" href="#">
-                    Search
-                  </a>
-                </div>
-              </div>
-            </div>
+        <!-- Navigation button -->
+        <button class="button button__cta" href="#">Post a vacancy</button>
+      </div>
+    </header>
 
-            <!-- Right wrapper -->
-            <div class="right-wrapper">
-              <img
-                class="right-wrapper__img"
-                src="./src/img/satisfied-lovely-woman-holds-modern-cell-phone-1.png"
-                alt="satisfied-lovely-woman-holds-modern-cell-phone"
-              />
+    <main>
+      <!-- Hero section -->
+      <section class="hero">
+        <!-- Hero section wrapper -->
 
-              <!-- Big ellipse -->
-              <img
-                class="right-wrapper__ellipse-big"
-                src="./src/img/ellipse-big.svg"
-                alt="big ellipse"
-              />
-              <!-- Small ellipse -->
-              <img
-                class="right-wrapper__ellipse-small"
-                src="./src/img/ellipse-small.svg"
-                alt="small ellipse"
-              />
+        <!-- Info -->
+        <div class="info">
+          <h1 class="info__title title">Find & Apply For Your Dream Job</h1>
+          <p class="info__subtitle">
```

Comment:
```suggestion
          <p class="subtitle">
```
The same. See my comment above.